tests/src/python/test_qgsserver_accesscontrol_wms.py

```diff
@@ -276,10 +276,31 @@ def test_wms_getmap_grp(self):
             headers.get("Content-Type"), "text/xml; charset=utf-8",
             f"Content type for GetMap is wrong: {headers.get('Content-Type')}")
         self.assertTrue(
-            str(response).find('<ServiceException code="Security">') != -1,
+            str(response).find('<ServiceException code="LayerNotDefined">') != -1,
             "Not allowed do a GetMap on Country_grp"
         )
 
+        # Check group ACL.
```

Comment:
Are the other cases described in the PR main description already tested? If not please add test cases or them.

Reply:
The other cases are already tested. The only one was not tested is when a group contains a restricted and not restricted layer.